### PR TITLE
TNO-1304: Table hover bug

### PR DESCRIPTION
**Pull request by jdtoombs**
**Blocked by #750 as I will have to update the NPM package but putting this up for review.**

Seems to have been some CSS that we don't need but wanted to confirm with you @Fosol. Rows still signify hover over but got rid of that weird behaviour from hovering the Group Title.

### CODE REVIEW

File: app/subscriber/tno-core-0.0.249.tgz
Comment (Fosol) [inline app/subscriber/tno-core-0.0.249.tgz:1]: Delete

File: app/subscriber/package.json
Comment (Fosol) [inline app/subscriber/package.json:55]: Fix

**Review by Fosol — CHANGES_REQUESTED** [review]
I think you're waiting until your other PR is merged before publishing a new package.

**Comment by jdtoombs** [comment]
@Fosol - updated

**Review by Fosol — APPROVED** [review]

**Pull request merged**
